app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -64,4 +66,186 @@ class SmokeTest {
             }
         }
     }
-}
+
+    @Test
+    fun verifyPageMainMenuItemsListInPortraitNormalMode() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        homeScreen {
+            navigationToolbar {
+            }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+            }.openThreeDotMenu {
+                verifyThreeDotMainMenuItems()
+            }.openHistory {
+                verifyTestPageUrl(defaultWebPage.url)
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openBookmarks {
+                verifyBookmarksMenuView()
+                verifyEmptyBookmarksList()
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openSettings {
+                verifySettingsView()
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openFindInPage {
+                verifyFindInPageSearchBarItems()
+            }.closeFindInPage {
+            }.openThreeDotMenu {
+            }.addToFirefoxHome {
+                verifySnackBarText("Added to top sites!")
+            }.openTabDrawer {
+            }.openHomeScreen {
+                verifyExistingTopSitesTabs(defaultWebPage.title)
+            }.openTabDrawer {
+            }.openTab(defaultWebPage.title) {
+            }.openThreeDotMenu {
+            }.openAddToHomeScreen {
+                verifyShortcutNameField(defaultWebPage.title)
+                clickAddShortcutButton()
+                clickAddAutomaticallyButton()
+                verifyShortcutIcon()
+            }.openHomeScreenShortcut(defaultWebPage.title) {
+            }.openThreeDotMenu {
+            }.bookmarkPage {
+                verifySnackBarText("Bookmark saved!")
+            }.openThreeDotMenu {
+            }.sharePage {
+                // verifyShareAppsLayout()
+                // verifyShareOverlay()
+            }.closeShareDialogReturnToPage {
+            }.openThreeDotMenu {
+            }.refreshPage {
+                // verifyPageContent(defaultWebPage.content)
+            }
+        }
+    }
+
+        @Test
+        fun verifyPageMainMenuItemsListInPortraitPrivateMode() {

```

Comment:
Just realized that the test title should end with Test..`verifyPageMainMenuItemsListInPortraitPrivateModeTest`

Reply:
okay